### ISSUE TITLE
Twombli not recognizing dominant direction results window

### ISSUE THREAD

**Issue by bmburkel** [wershofe/TWOMBLI#8]
Hi. I have been a user of ctFIRE for fiber analysis, but I am trying to use your program to analyze a picrosirius red stained TMA. Unfortunately, I am having some difficulties. I have read all the instructions and watched your youtube videos. I can get everything set up, and the early parts of the pipelines work fine. I run into problems when I try to complete the analysis on both the training sets and my data. After the orientationJ plugin is called, a "Results Orientation" window pops up. An error window then appears saying it can't find "Dominant Direction of "x".png. When I look at the code it seems to be referring to lines 1256-1258.  I don't see anything specifically wrong about those lines, but it doesn't work. If I try to troubleshoot it with a print call with a getResult("Coherency [%]", 0), it does not return anything. Has anybody seen a problem like this and is there a work around? Both my orientationJ and FIJI should be up to date. When I update it brings me to version 1.54f. I am very eager to use your workflow and any assistance you can provide would be greatly appreciated.
![tutorial error Capture](https://github.com/wershofe/TWOMBLI/assets/142931293/bcf68b62-cf79-493d-b2cc-840cc582fbc6)

**Comment by wershofe** [comment]
Hi
Have you been able to run the tutorial successfully? Could you check running the code on some other of your images in case it was one particular image causing the problem?

**Comment by bmburkel** [comment]
Yes, I did the tutorial. Twombli did not work on the images provided by the tutorial or my own. Everything was configured as you described. Eventually, I figured out that the problem was not with Twombli, but with the orientationJ plugin. Even though I use FIJI and update frequently, the updated (newest) version of OrientationJ does not seem to work. It produces an error. When I went to the EPFL website and downloaded the 2.0.5 jar, it seems to work fine. So, the problem is somewhere in the OrientationJ plugin.

**Comment by wershofe** [comment]
OK thank you for pointing this out. We will add this to the documentation. We've just started rewriting Twombli V2.0 that will not have these kinds of dependencies, but that is probably a few months away
https://forum.image.sc/t/designing-twombli-v2-0/82130
